src/components/ToppingsSelect/index.jsx

```diff
@@ -3,14 +3,39 @@ import Topping from "../Topping";
 import './style.css';
 
 const ToppingsSelect = ({ toppings }) => {
+
+  const [toppingsToChoose, setToppingsToChoose] = useState(toppings);
+
+  const handleSelect = (index, isChecked) => {
+    const newToppings = [...toppings];
+    newToppings[index].selected = isChecked;
+    setToppingsToChoose(newToppings);
+      
+    };
+
+  const selectedToppings = toppingsToChoose.filter((topping) => topping.selected === true)
+
+  let totalCount = selectedToppings.length;
+  let totalPrice = 0;
+
+  selectedToppings.forEach((topping) => {
+    totalPrice += topping.price; 
+  });
```

Comment:
zvládneš to pomocí reduce? ;) https://stackoverflow.com/questions/46680151/using-reduce-to-get-sum-of-all-numbers-in-an-array-in-javascript

Reply:
Prostuduju a zkusím to :-)